### DISCUSSION TITLE
About assignment due date. [I want to deliver early.]

### DISCUSSION THREAD

**Discussion by muaz742** [skills/.github#160] [Q&A]
Hi 👋🏻

I have completed my assignment. 
(All my action process are blue ticked.) 

https://user-images.githubusercontent.com/39378729/216600637-3055e777-6ab6-4fc3-8e38-289a8b6a5bf1.mov

But my delivery date is in a few months. 

<img width="528" alt="2023-02-03 15 04 27" src="https://user-images.githubusercontent.com/39378729/216600662-315d7b5c-aad7-4c5e-bf1b-f90f164baffd.png">

Is there a way to deliver early?

**Comment by SlimBloodworth** [comment]
Does it have an option to submit early? It's saying the assignment is due **_BY_** June 30, 2023... There should be and option to submit at any point before that date.

**Reply by muaz742** [reply]
I agree you but I can't find this early submit way...

**Reply by markpatterson27** [reply]
There is no 'submit' option. As long as your commits are in the assignment repo on GitHub it's considered 'submitted'. GitHub Classroom will link to the last commit before the assignment deadline when it's reviewed by your 'teacher'/assessor.
